CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
```

Comment:
can we update changelog ? It captured all changes since v14.0
cc: @saikumarrs

Reply:
Updated the changelog